docs/docsite/rst/dev_guide/collection_requirements.rst

```diff
@@ -0,0 +1,437 @@
+  * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
+
+* You MUST run CI against each of the "major versions" (2.10, 2.11, 2.12, etc) of ``ansible-base``/``ansible-core`` that the collection supports. (Usually the ``HEAD`` of the stable-xxx branches.)
+* All CI tests MUST run against every pull request and SHOULD pass before merge.
+* At least sanity tests MUST run against a commit that releases the collection; if they do not pass, the collection will NOT be released.
+
+  - If the collection has integration/unit tests, they SHOULD run too; if they do not pass, the errors SHOULD be analyzed to decide whether they should block the release or not.
+* All CI tests MUST run regularly (nightly, or at least once per week) to ensure that repositories without regular commits are tested against the latest version of ansible-test from each ansible-base/ansible-core version tested. The results from the regular CI runs MUST be checked regularly.
+
+All of the above can be achieved by using the `GitHub Action template <https://github.com/ansible-collections/collection_template/tree/main/.github/workflows>`_.
+
+To learn how to add tests to your collection, see:
+
+* `Quick-start integration testing guide <https://github.com/ansible/community-docs/blob/main/integration_tests_quick_start_guide.rst>`_
+* `Quick-start unit testing guide <https://github.com/ansible/community-docs/blob/main/unit_tests_quick_start_guide.rst>`_
+
+Collections and Working Groups
+==============================
+
+The collections have:
+
+* Working group page(s) on a corresponding wiki if needed. Makes sense if there is a group of modules for working with one common entity, for example postgresql, zabbix, grafana, and so on.
+* Issue for agenda (or pinboard if there are not regular meetings) as a pinned issue in the repository.
+
+When moving modules between collections
+=======================================
+
+All related entities must be moved/copied including:
+
+* Related plugins and module_utils files (when moving, be sure it is not used by other modules, otherwise copy).
+* CI and unit tests.
+* Corresponding documentation fragments from ``plugins/doc_fragments``.
+
+Also:
+
+* Change ``M()``, examples, ``seealso``, ``extended_documentation_fragments`` to use actual FQCNs in moved content and in other collections that have references to the content.
+* Move all related issues, pull requests, and wiki pages.
+* Look through ``docs/docsite`` directory of `ansible-base GitHub repository <https://github.com/ansible/ansible>`_ (for example, using the ``grep`` command-line utility) to check if there are examples using the moved modules and plugins to update their FQCNs.
+
+See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
+
```

Comment:
```suggestion

.. _collection_development_conventions:

```

Reply:
This fix attempts to clear up the rst warning shown by CI for duplicate explicit target name.